packages/block-library/src/cover/block.json

```diff
@@ -40,6 +40,9 @@
 		},
 		"customGradient": {
 			"type": "string"
+		},
+		"padding": {
```

Comment:
Perhaps we could follow suit on what was done for colors & font-size, so we wouldn't need to declare this attribute per block and declaring support for `__experimentalPadding` would suffice.